### ISSUE TITLE
The imports in the RxFire Auth docs should refer to the auth module

### ISSUE THREAD

**Issue by dungahk** [firebase/firebase-js-sdk#1292]
#### Steps to reproduce:
Access the RxFire Auth docs https://github.com/firebase/firebase-js-sdk/blob/master/packages/rxfire/docs/auth.md

Example:
Instead of 
```ts
import { authState } from 'rxfire/firestore';
```

It should be 
```ts
import { authState } from 'rxfire/auth';
```

All imports seem to be referring to `firestore` instead of `auth`.

**Comment by Feiyang1** [comment]
Hi @dungahk, Thanks for pointing it out and PR is welcomed!

**Comment by davideast** [comment]
@dungahk I would totally love an accept a PR here!

**Comment by dungahk** [comment]
Done!
Btw, great job guys, RxFire is awesome!

**Comment by dungahk** [comment]
Closed by #1297
